src/appleseedmaya/typeids.h

```diff
@@ -109,6 +109,7 @@ enum AppleseedMayaTypeIds
     AsSbsPbrMaterialNodeTypeID      = 0x00127a04,   // 1210884
     AsManifold2DNodeTypeID          = 0x00127a05,   // 1210885
     AsManifold3DNodeTypeID          = 0x00127a06,   // 1210886
+    AsAscCdlNodeTypeId              = 0x00127a07,   // 1210887
```

Comment:
While you're modifying this file, could you replace the `ID` suffixes by `Id` to make naming consistent?  Indentation also needs to be fixed.

Reply:
Id suffixes corrected. But I can't see what is the issue with the indentation

Reply:
Thanks!

The indentation "problem" was simply a glitch of GitHub's UI (maybe a small regression on their side, I haven't seen this before):

![image](https://user-images.githubusercontent.com/321290/50219279-00caad00-038f-11e9-928f-aca0d94a1400.png)

Reply:
Don't the renamings affect other code?

Reply:
Apparently not... Merging!